server/inngest/crons/upcomingEventNotifications.ts

```
@@ -0,0 +1,31 @@
+
+export default inngest.createFunction(
+  {
+    name: "Find Events To Send Upcoming Notifications",
+    id: "find-events-to-send-upcoming-notifications",
+  },
+  {
+    cron: "0 16 * * *", // every day at 1 PM Brazil time (16:00 UTC)
+  },
+  async () => {
+    const events = await getEventsToSendUpcomingNotifications();
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)